scopes/component/ui/component-meta/component-overview.tsx

```diff
@@ -34,7 +41,10 @@ export function ComponentOverview({
   return (
     <Section {...rest}>
       <div className={textColumn}>
-        <H1>{displayName}</H1>
+        <div className={styles.componentTitle}>
+          <H1>{displayName}</H1>
+          {titleBadges?.map((titleBadge) => titleBadge.component())}
```

Comment:
render this is a react component:
```suggestion
          {titleBadges?.map((titleBadge) => <titleBadge.component />)}
```

also - what about the Component prop?